http-driver/src/main/java/org/neo4j/ogm/drivers/http/response/RestModelAdapter.java

```diff
@@ -62,8 +62,8 @@ private Object processData(Object element) {
 	private Object buildEntity(Map entity) {
 		if (entity.containsKey("metadata") && ((Map) entity.get("metadata")).get("id") != null) {
 			Map entityMetadata = (Map) entity.get("metadata");
-			if (entityMetadata.containsKey("labels")) {
-				List<String> labelList = (List<String>) (entityMetadata.get("labels"));
+			if (entityMetadata.containsKey("addedLabels")) {
```

Comment:
Is there a reason the key name has changed? Is it simply to distinguish between addedLabels and removedLabels?